jest.config.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+  transform: {
+    '^.+\\.(t|j)sx?$': '@swc/jest',
+  },
+  testEnvironment: 'jsdom',
+  testRegex: 'tests/.*test\\.tsx?$',
+  collectCoverage: true,
```

Comment:
would it make sense to add coverage threshold so we make sure any changes added will have tests and coverage?

Reply:
I think we need to integrate coveralls first and set a coverage threshold from there. I will add this issue to the DevOps backlog